src/comment.cpp

```diff
@@ -304,7 +304,7 @@ void CommentParser::collectParameters(const char *fulltext, FileModule *root_mod
 
 		// Look for the group to which the given assignment belong
 		int i=0;
-		for (;i<groupList.size() && groupList[i].lineNo<firstLine;i++);
+		for (; (unsigned)i < groupList.size() && groupList[i].lineNo<firstLine;i++);
```

Comment:
@amarjeetkapoor1 You can apply this on your own branch if you want. We just merged this without as it was supposed to be some quick fixes to remove warning, not smth. that requires a conversation..